internalsdk/android_test.go

```diff
@@ -160,7 +163,7 @@ func testProxiedRequest(helper *integrationtest.Helper, proxyAddr string, dnsGra
 
 	var buf []byte
 
-	buf, err = ioutil.ReadAll(res.Body)
+	buf, err = io.ReadAll(res.Body)
```

Comment:
Can you add a sanity check to the length of the body here? Just check Response.ContentLength or whatever it's called, or add a io.LimitReader to some reasonably (large) value? Although I do see a timeout above, if it applies to body reading then it's moot.

Reply:
It thing it applies to the entire request, including reading the body.